### DISCUSSION TITLE
Display item count in sidebar

### DISCUSSION THREAD

**Discussion by rasco** [directus/directus#18893] [Feature Requests]
### Summary

I think it would be helpful if you could opt-in to show the item count in the sidebar for collections and bookmarks.

This would be especially helpful for bookmarks. 
In our case, we heavily use global bookmarks to allow quick access to certain items (filtered). 

### Basic Example

For instance: A bookmark that shows Articles ready for review.
If the (filtered) item count was shown in the sidebar, Editors would immediately know if there are any items to review, and how many. It would prevent having to open each bookmark and check if there are any items.

### Motivation

It would speed up some content creation processes because you wouldn't need to open the entire collection to see how many items are in there.

### Detailed Design

It could look something like this:
![](https://cdn.discordapp.com/attachments/741317677397704757/1116624213516763136/Screenshot_2023-06-09_at_3.05.44_PM.png)

### Must Have:
- Number of total (filtered) items beside a collection in the sidebar

### Should Have:
- Automatic refresh
- Lazy loading (for performance reasons)

### Could Have:
- Ability to opt-out (for performance reasons)

### Won't Have:
- Folders don't need to show an item count

### Drawbacks

Performance could be heavily impacted because some systems might have a huge number of collections. 
That's why lazy loading is probably unavoidable. 

### Alternatives

Alternatively, the item count could be shown while hovering over the collection

### Adoption Strategy

Should have no adoption impacts

### Unresolved Questions

How would the numbers be refreshed?
Maybe through a Websocket connection?
Is polling realistic performance-wise? Probably not

**Comment by JonL1** [comment]
Thanks for doing this. I never came around of opening a FR for it but it’s a feature that is quite standard in other internal tools.

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by DanielBiegler** [comment]
Thanks for the request! Just wanted to add a little footnote that in the meantime without this feature, building a dashboard via the Insights-Module could improve your workflow. The dashboard can show multiple review-queues depending on your filters and show big bright numbers so you can get an overview in a single page instead of opening multiple bookmarks. :smile:

**Comment by lahdekorpi** [comment]
Has this idea been buried?

**Reply by rijkvanzanten** [reply]
If by _buried_ you mean _other things take priority over this_ then yes! We’ve also moved where we’re tracking feature requests to help make sure things don’t get stale over here. See https://github.com/directus/directus/discussions/25951 For more info on that :)

**Reply by JamesW1** [reply]
Rest assured it's tracked in our improvements backlog internally and I will provide an update on this thread when we get to it 🙂
